### PR TITLE
Add fullscreen popup OT article

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add fullscreen popup OT article.

### CODE REVIEW

File: site/en/blog/fullscreen-popups-origin-trial/index.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+## Demo
+
+Try fullscreen popup windows in the
+[demo](https://michaelwasserman.github.io/window-placement-demo/) on Glitch and
+[view the source code](https://github.com/michaelwasserman/window-placement-demo/).
+Be sure to check the **fullscreen** checkbox, and, if you have the opportunity,
+play with the demo with multiple screens attached to your device.
+
+<iframe src="https://michaelwasserman.github.io/window-placement-demo/" allow="window-management"></iframe>
```
Comment (tomayac) [inline site/en/blog/fullscreen-popups-origin-trial/index.md:101]:
```suggestion
<iframe src="https://michaelwasserman.github.io/window-placement-demo/" style="width: 100%; height: 800px;" allow="window-management"></iframe>
```

**Review by rachelandrew — APPROVED** [review]

File: site/en/blog/fullscreen-popups-origin-trial/index.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+​​It's not required to add `popup` in the `windowFeatures` strings here, as it's
+implicitly assumed when any of `left`, `top`, `width`, or `height` is defined.
+{%endAside %}
+
+## Demo
+
+Try fullscreen popup windows in the
+[demo](https://michaelwasserman.github.io/window-placement-demo/) and
+[view the source code](https://github.com/michaelwasserman/window-placement-demo/).
+Be sure to check the **fullscreen** checkbox and **Fullscreen popup** button, and,
```
Comment (tomayac) [inline site/en/blog/fullscreen-popups-origin-trial/index.md:98]:
```suggestion
Be sure to check the **fullscreen** checkbox and the **Fullscreen popup** button, and,
```

**Review by michaelwasserman — COMMENTED** [review]
lgtm, thanks for the updates! If it's not too much trouble, please also add me to the reviewers acknowledgements, thanks! `[Mike Wasserman](https://www.linkedin.com/in/mike-wasserman)`

File: site/en/blog/fullscreen-popups-origin-trial/index.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+- [Chromium bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1142516)
+- [TAG review](https://github.com/w3ctag/design-reviews/issues/840)
+- [Mozilla standards position](https://github.com/mozilla/standards-positions/issues/714)
+- [WebKit standards position](https://github.com/WebKit/standards-positions/issues/101)
+
+## Acknowledgements
+
+This article was reviewed by [Brad
+Triebwasser](https://www.linkedin.com/in/bradtriebwasser), [Hakan
+Isbiliroglu](https://www.linkedin.com/in/hakan-isbiliroglu), and [Rachel
```
Comment (tomayac) [inline site/en/blog/fullscreen-popups-origin-trial/index.md:117]:
```suggestion
Isbiliroglu](https://www.linkedin.com/in/hakan-isbiliroglu),
[Mike Wasserman](https://www.linkedin.com/in/mike-wasserman), and [Rachel
```

**Comment by tomayac** [comment]
Review acknowledged in https://github.com/GoogleChrome/developer.chrome.com/pull/7374/commits/a94985bbef478534cb7f8f6fa52eb83fc1014995. Thanks, Mike!

**Pull request merged**